### PR TITLE
update version

### PR DESCRIPTION
**Pull request by dimbleby**
It would have been more helpful if I'd noticed this yesterday, wouldn't it?

### CODE REVIEW

**Comment by dimbleby** [comment]
(perhaps we should add something to the pipeline to verify that this version matches the version as defined elsewhere)

**Comment by abn** [comment]
We should really drop `__version__`.

File: src/poetry/core/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 
 
-__version__ = "1.1.0a7"
+__version__ = "1.1.0b1"
```
Comment (abn) [inline src/poetry/core/__init__.py:8]:
```suggestion
```

We have been promoting the use of importlib elsewhere for this use case. We should really dogfood our own advice on the matter and either remove this or use importlib/pkgutil.

Comment (dimbleby) [reply src/poetry/core/__init__.py:8]: I'm not fussed either way - feel free either to update this MR or to reject it in favour of something else

Comment (abn) [reply src/poetry/core/__init__.py:8]: Let's see what the rest of the team thinks. My vote is to just remove it.

Comment (neersighted) [reply src/poetry/core/__init__.py:8]: Let's just drop it.

File: src/poetry/core/__init__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
```
Comment (abn) [inline src/poetry/core/__init__.py:9]:
```suggestion
```

File: src/poetry/core/__init__.py (hunk 1 above)
Comment (abn) [reply src/poetry/core/__init__.py]: Hmm, apparently not. We imprint the metadata for wheels with the version it seems.

**Review by neersighted — APPROVED** [review]

**Comment by abn** [comment]
@dimbleby sorry for the noise.

**Pull request closed without merging**